src/main.js

```diff
@@ -435,7 +435,7 @@ Formsy.Form = React.createClass({
   render: function () {
 
     return (
-      <form {...this.props} onSubmit={this.submit}>
+      <form id={this.props.id} {...this.props} onSubmit={this.submit}>
```

Comment:
It seems that it was not included in the release.
You can use the latest version in this way:
```
npm install --save christianalfoni/formsy-react
```

@christianalfoni how about to create new 0.14.x version before 0.15.0 ?